packages/flutter/lib/src/widgets/semantics_debugger.dart

```diff
@@ -365,7 +365,10 @@ class _SemanticsDebuggerPainter extends CustomPainter {
     }
     final Rect rect = node.rect;
     if (!rect.isEmpty) {
-      final Color lineColor = Color(0xFF000000 + math.Random(node.id).nextInt(0xFFFFFF));
+      // To minmize changes to goldens tests, use a static color.
+      final Color lineColor = const bool.fromEnvironment('FLUTTER_TEST')
```

Comment:
Who provides this dart define?

Reply:
Or was this supposed to be from an environment variable?

Reply:
Your right, I didn't know this was for dart defines.

Is there an agnostic way to do this in the framework? Pulling in dart:io here would be a blocker

Reply:
Good point, we probably can't make this change in the framework...